### DISCUSSION TITLE
Education benefit is not received after one week of approval

### DISCUSSION THREAD

**Discussion by YudanYing** [community/community#131140] [GitHub Education]
### Hi everyone,

It is shown that my application for GitHub Education was approved on Jun 27, but I did not receive any confirmation email nor did I received the education benefit. Would you help to see what is going on? 

Thanks!

**Comment by gride29** [comment]
Same thing - GitHub Copilot stopped working.

Edit

@YudanYing If you want to use Github Copilot - you just need to go to right upper panel (click on your avatar) and select "Your Copilot" then you will be able to activate it. I also didn't get confirmation email, but it works like that.

**Comment by ChunChih3310** [comment]
Same issue. Additionally, there is a message on the signup page that states, "Accounts that participated in the year-long Technical Preview of GitHub Copilot are not eligible for the 30-Day Free Trial." I am unsure if this is the reason I am unable to access Copilot for free as a student.
![image](https://github.com/community/community/assets/90191280/e8440332-7969-4a17-b3bb-cf36cef648d0)

**Reply by gride29** [reply]
Check my answer.

**Reply by ChunChih3310** [reply]
But this is already a screenshot of me clicking "Your Copilot" and it still hasn't activated successfully.

**Comment by YudanYing** [comment]
I got an email this morning showing that I'm part of GitHub Education, and then the Copilot works again. I guess you just need to either wait or contact them for the approval/activation of your Education account.
